src/groovy/org/pih/warehouse/core/ActivityCode.groovy

```diff
@@ -64,6 +64,8 @@ enum ActivityCode {
 
     AUTOSAVE('AUTOSAVE'),
 
+    REQUEST_APPROVAL('REQUEST_APPROVAL'),
```

Comment:
Why can't we use APPROVE_REQUEST above?

Reply:
Honestly I have not seen this before. I checked it and it seems to be existing, but not having any effect.
It's also displayed as `Approve requisition` - I saw your comments in @alannadolny 's ticket and I am a bit confused. In tickets it was supposed to be named with "Request" and you suggest that it should be "Requisition" instead of "Request".
So would it mean this activity code should be also renamed from `APPROVE_REQUEST` to `APPROVE_REQUISITION`?

I think we should not mix "Requisition" and "Request", so the question is, what is the path we want to follow?